src/charm.lua

```diff
@@ -816,7 +844,7 @@ Bakery_API.Charm {
         }
     end,
     calculate = function(self, card, context)
-        if context.individual and context.cardarea == 'unscored' then
+        if not card.debuff and context.individual and context.cardarea == 'unscored' then
```

Comment:
This is perhaps the best argument for actually debuffing the charms. Maybe do both, since it'd be weird the check `G.GAME.Bakery_charm` here.